iina/AppData.swift

```diff
@@ -93,6 +93,7 @@ struct Constants {
     static let keyBindingInputChanged = Notification.Name("IINAkeyBindingInputChanged")
     static let fileLoaded = Notification.Name("IINAFileLoaded")
     static let historyUpdated = Notification.Name("IINAHistoryUpdated")
+    static let legacyFullScreen = Notification.Name("IINALegacyFullScreen")
```

Comment:
Not completely related, but I think this struct would be better if it was an extension of `Notification.Name`.